packages/country-config/src/transform.ts

```diff
@@ -39,6 +39,13 @@ export interface MOSIPPayload {
     deathDeclared: string;
     dateOfDeath: string;
   };
+  metaInfo: {
+    metaData: string;
+    registrationId: string;
+    operationsData: string;
+    capturedRegisteredDevices: string;
+    creationDate: string;
+  };
```

Comment:
@tharakadadigama20 just thinking if it would be better if the meta data had a separate type.  Then we could stringify here like before... because this approach might make us validate the string if it actually is a valid JSON or not. Besides, a country implementor might be unsure what string to put here. Thoughts?